measure/src/main/scala/org/apache/griffin/measure/step/builder/dsl/transform/CompletenessExpr2DQSteps.scala

```diff
@@ -85,8 +85,16 @@ case class CompletenessExpr2DQSteps(context: DQContext,
 
       // 2. incomplete record
       val incompleteRecordsTableName = "__incompleteRecords"
-      val completeWhereClause = aliases.map(a => s"`${a}` IS NOT NULL").mkString(" AND ")
-      val incompleteWhereClause = s"NOT (${completeWhereClause})"
+      val errorConfs: Seq[RuleErrorConfParam] = ruleParam.getErrorConfs
+      var incompleteWhereClause: String = ""
+      if (errorConfs.size == 0) {
+        // old completeness way
+        val completeWhereClause = aliases.map(a => s"`${a}` IS NOT NULL").mkString(" AND ")
+        incompleteWhereClause = s"NOT (${completeWhereClause})"
+      } else {
+        // new completeness way
```

Comment:
can we comment here as "with errorconf".

Reply:
ok, I will update these comments soon.

This is my first Griffin pr, and I find the CI failed. However, I cannot search the Failures or Errors test in the log. Could you please help me find why the CI failed? Thank you.

Reply:
let's figure out why it failed.

Reply:
update the comment